grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -442,6 +456,91 @@ public GrpcServiceBuilder useClientTimeoutHeader(boolean useClientTimeoutHeader)
         return this;
     }
 
+    /**
+     * Sets the specified {@link GrpcStatusFunction} that maps a {@link Throwable} to a gRPC {@link Status}.
+     * The mapping function are used to handle a {@link Throwable} when it is raised.
```

Comment:
nit: is used?
Or, I think we can just remove this line. 🤔

Reply:
Removed